src/Surfnet/ServiceProviderDashboard/Application/Service/EntityService.php

```diff
@@ -41,12 +41,26 @@ class EntityService implements EntityServiceInterface
      */
     private $router;
 
+    /**
+     * @var string
+     */
+    private $oidcPlaygroundUriTest;
+
+    /**
+     * @var string
+     */
+    private $oidcPlaygroundUriProd;
+
     public function __construct(
         EntityQueryRepositoryProvider $entityQueryRepositoryProvider,
-        RouterInterface $router
+        RouterInterface $router,
+        $oidcPlaygroundUriTest,
+        $oidcPlaygroundUriProd
     ) {
         $this->queryRepositoryProvider = $entityQueryRepositoryProvider;
         $this->router = $router;
+        $this->oidcPlaygroundUriTest = $oidcPlaygroundUriTest;
```

Comment:
As these values are injected from the parameters.yml, it's a good idea to perform input validation on them.